contracts/Splitter.sol

```diff
@@ -18,32 +22,44 @@ contract Splitter {
         uint amount
     );
 
-    function splitDeposit(address payable recipient1, address payable recipient2) external payable{
+    constructor() Pausable() public {}
+
+    function splitDeposit(address payable recipient1, address payable recipient2) external payable whenNotPaused {
```

Comment:
Where is `recipient1` sent any Ether, in this function?

Reply:
My understanding of the best practice, based on the code review with Rob, is that the function should not both receive and send ether. Instead the contract should be the only recipient of ether. I should then maintain a balance of the ether owed to the recipient by the contract.

`balances[recipient1] = SafeMath.add(balances[recipient1], split);`

Then recipient1 can then subsequently withdraw Ether owed.

Reply:
That is correct.
But by saying `address payable recipient1`, you inform me that `recipient1` is going to have `.transfer` called on it within the scope of the function... This is incorrect.

Reply:
I understand. I have removed the payable designation.